README.md

```diff
@@ -58,6 +58,37 @@ sudo make install
 # install fzf: https://github.com/junegunn/fzf
 ```
 
+### Using oh-my-zsh
+
+Make sure that your oh-my-zsh `ZSH_CUSTOM` directory is configured,
+then clone navi into the plugins directory.
+
+```sh
+export ZSH_CUSTOM='/path/to/.zsh'
+
+DST="$ZSH_CUSTOM/plugins"
+mkdir -p "$DST" && cd "$DST"
+git clone http://github.com/denisidoro/navi
+```
+
+To enable navi, simply add it to the oh-my-zsh plugin array:
+
+```
+# Sample ~/.zshrc
+ZSHCFG="$HOME/.zsh"
+ZSH="$ZSHCFG/oh-my-zsh"
+ZSH_CUSTOM="$ZSHCFG"
+plugins=(archlinux docker compleat git github gpg-agent ssh-agent nvm tmux fzf workon navi)
```

Comment:
`plugins=(docker tmux navi)`, just to make it more succint

Reply:
Oops, I copied from my `.zshrc` but forgot to trim the output. Good call.